lib/src/Uncertainty/Algorithm/Optimization/EfficientGlobalOptimization.cxx

```diff
@@ -379,6 +379,21 @@ void EfficientGlobalOptimization::run()
     }
 
     ++ iterationNumber;
+
+    // callbacks
+    if (progressCallback_.first)
+    {
+      progressCallback_.first((100.0 * iterationNumber) / getMaximumIterationNumber(), progressCallback_.second);
+    }
+    if (stopCallback_.first)
+    {
+      Bool stop = stopCallback_.first(stopCallback_.second);
+      if (stop)
+      {
+        convergence = true;
```

Comment:
That looks ugly, please rename the 'convergence' variable, eg. exitLoop or similar.